docs/tests/integration/test_logcollector/test_location/index.md

```diff
@@ -0,0 +1,26 @@
+# Test location
+
+## Overview 
+
+These tests check if the introduced configuration works as expected for valid values and if the logcollector 
+response for analyzing requests works properly.
+
+## Objective
+
+Confirm that the different options for logcollector configuration work and are correctly loaded.
+
+## General info
+
+|Tier | Number of tests | Time spent |
+|:--:|:--:|:--:|
+| 0 | 21 | 42.20s |
+
+## List of configuration tests
+
+- **[Test location](test_location.md)**: Check if logcollector works using different `location` values.
+
+- **[Test exclude](test_exclude.md)**: Check if logcollector works using different `location` and `exclude` values.
+
+
+
+
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/8dd2eb232ec5e6ed9c8b14ba1437da2b51a77b63